### PR TITLE
add fs processor to own nar

### PR DESCRIPTION
**Pull request by jeffklassen**

### CODE REVIEW

File: geomesa-nifi-processors/geomesa-nifi-processors-fs/pom.xml [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+
+        <dependency>
+            <groupId>org.geomesa.nifi</groupId>
+            <artifactId>geomesa-nifi-controllers-api</artifactId>
+        </dependency>
+
+        <dependency>
+        <dependency>
+            <groupId>org.locationtech.geomesa</groupId>
+            <artifactId>geomesa--datastore_2.11</artifactId>
```
Comment (elahrvivaz) [inline geomesa-nifi-processors/geomesa-nifi-processors-fs/pom.xml:43]: looks like a typo here

Comment (jeffklassen) [reply geomesa-nifi-processors/geomesa-nifi-processors-fs/pom.xml]: great call, fixed.

File: geomesa-nifi-processors/geomesa-nifi-processors-fs/pom.xml [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+        </dependency>
+
+        <dependency>
+            <groupId>org.apache.zookeeper</groupId>
+            <artifactId>zookeeper</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>org.apache.accumulo</groupId>
+            <artifactId>accumulo-core</artifactId>
```
Comment (elahrvivaz) [inline geomesa-nifi-processors/geomesa-nifi-processors-fs/pom.xml:56]: we shouldn't need accumulo in this nar, can you remove this?

File: geomesa-nifi-processors/geomesa-nifi-processors-fs/src/main/services/org.apache.nifi.processor.Processor [hunk 3]
```diff
@@ -0,0 +1,7 @@
+org.geomesa.nifi.processors.datastore.PutGeoTools
```
Comment (elahrvivaz) [inline geomesa-nifi-processors/geomesa-nifi-processors-fs/src/main/services/org.apache.nifi.processor.Processor:1]: this file can be deleted, should just need the one under META-INF

**Comment by elahrvivaz** [comment]
merged as d53c7aa0cdb21e95376beb0b31014f238664b2ae

**Pull request closed without merging**